cmake/afr.cmake

```diff
@@ -15,6 +15,19 @@ set(AFR_DEMOS_DIR "${AFR_ROOT_DIR}/demos/common" CACHE INTERNAL "Amazon FreeRTOS
 set(AFR_TESTS_DIR "${AFR_ROOT_DIR}/tests/common" CACHE INTERNAL "Amazon FreeRTOS tests root.")
 set(AFR_3RDPARTY_DIR "${AFR_MODULES_DIR}/third_party" CACHE INTERNAL "3rdparty libraries root.")
 
+# These paths will be used for boards that support BLE.
+set(BLE_LIBRARY_INCLUDE_DIR "${AFR_MODULES_DIR}/include/bluetooth_low_energy" CACHE INTERNAL "BLE include directory.")
+set(BLE_DEMO_COMMON_SOURCES "${AFR_DEMOS_DIR}/ble/iot_ble_numericComparison.c" CACHE INTERNAL "BLE demo common source files.")
+
+# Network manager will temporarily be built into the demo executables.
+set(NETWORK_MANAGER_SOURCES
```

Comment:
So this will be refactored to a demo in the future?

Reply:
It will be a separate library in the future.